v4/newrelic/transaction.go

```diff
@@ -198,7 +202,11 @@ func (txn *Transaction) StartSegment(name string) *Segment {
 //
 // StartExternalSegment calls InsertDistributedTraceHeaders, so you don't need
 // to use it for outbound HTTP calls: Just use StartExternalSegment!
-func (txn *Transaction) InsertDistributedTraceHeaders(hdrs http.Header) {}
+func (txn *Transaction) InsertDistributedTraceHeaders(hdrs http.Header) {
+	props := propagation.New(propagation.WithInjectors(trace.TraceContext{}))
```

Comment:
Is this something we can cache on the txn itself?

Reply:
Or even the app???

Reply:
Good point, I put it on the app.